src/uniparc/components/landing-page/styles/landing-page.module.scss

```
@@ -0,0 +1,83 @@
+@import 'franklin-sites/src/styles/settings';
+@import 'franklin-sites/src/styles/colours';
+
+.landing-page {
+  margin-block: 2 * $global-margin;
+
+  p:last-of-type {
+    margin-bottom: 0;
+  }
+
+  table {
+    margin-bottom: $global-margin;
+    border-spacing: 0;
+    overflow: auto;
+
+    caption {
+      inline-size: max-content;
+    }
+  }
+
+  thead {
+    color: $colour-sky-white;
+    background: $colour-sea-blue;
+
+    th {
+      width: 50%;
+    }
+
+    a {
+      color: $colour-sky-white;
+    }
+  }
+
+  tbody {
+    border: 1px solid $colour-sea-blue;
+  }
+
+  th,
+  td {
+    padding: $global-padding * 0.25 $global-padding * 0.5;
+    border: 1px solid $colour-sea-blue;
+    border-collapse: collapse;
+  }
+}
+
+.table-container {
+  display: grid;
```

```suggestion
  margin-top: 2rem;
  display: grid;
```
To line up the table with the text in the `Searching the Database` and `Download` text to the right.